src/components/GiveProject/__tests__/GiveProject.unit.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { DecimalBigNumber } from "src/helpers/DecimalBigNumber/DecimalBigNumber";
+import { useCurrentIndex } from "src/hooks/useCurrentIndex";
+
+import { act, render } from "../../../testUtils";
+import ProjectCard, { ProjectDetailsMode } from "../ProjectCard";
+
+jest.mock("src/hooks/useCurrentIndex");
+
+describe("<ProjectCard />", () => {
```

Comment:
This is redundant because Brightiron added project card tests in the Give view test set